src/pages/SwapPage/SwapPage.tsx

```diff
@@ -55,50 +69,101 @@ const SwapPage: React.FC = () => {
 
   return (
     <Box width='100%' mb={3} id='swap-page'>
-      <Box
-        mb={2}
-        display='flex'
-        alignItems='center'
-        justifyContent='space-between'
-        width='100%'
-      >
-        <Typography variant='h4'>Swap</Typography>
-        <Box className={classes.helpWrapper}>
-          <Typography variant='body2'>Help</Typography>
-          <HelpIcon />
-        </Box>
-      </Box>
-      <Grid container spacing={4}>
-        <Grid item xs={12} sm={12} md={5}>
-          <Box className={classes.wrapper}>
-            <SwapMain />
+      {!isProMode && (
+        <Box
+          mb={2}
+          display='flex'
```

Comment:
We should start using css files instead of declaring everything online but that would be part of a larger refactor

Reply:
I think we can use scss and we can do this in the future